pkgs/top-level/all-packages.nix

```diff
@@ -2128,8 +2128,7 @@ with pkgs;
 
   espanso-wayland = espanso.override {
     x11Support = false;
-    waylandSupport = true;
-    espanso = espanso-wayland;
+    waylandSupport = !stdenv.hostPlatform.isDarwin;
```

Comment:
I don't think this can work, as Wayland support for espanso is not additive to but rather is mutually exclusive with X11 support.

Discussed in another PR that I don't have handy at the moment.

Reply:
Ah, I see. Hadn't thought that through, thanks.

Seems fine in that case.

I should should copy over the nixosTest I PRed to espanso/espanso. Those would run automatically when the module is changed, right?